modules/lima/meta.yml

```diff
@@ -0,0 +1,49 @@
+name: lima
+description: lima - The PacBio Barcode Demultiplexer and Primer Remover
+keywords:
+  - sort
+tools:
+  - lima:
+      description: lima - The PacBio Barcode Demultiplexer and Primer Remover
+      homepage: https://github.com/PacificBiosciences/pbbioconda
+      documentation: https://lima.how/
+      tool_dev_url: https://github.com/pacificbiosciences/barcoding/
+      doi: ""
+      licence: ['BSD-3-clause-Clear']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - ccs:
+      type: file
+      description: BAM file, consensus sequences produced by CCS
+      pattern: "*.bam"
```

Comment:
As mentioned above, it would be helpful if the other input types are supported too.

Reply:
By using the suffix option, i's possible to use bam, fasta, fasta.gz, fastq, fastq.gz. 
It's possible to use xml also, but would not encourage to use it. 
Those metadata files include full path to files, which, I think, could be problematic.